src/Raven.Server/Documents/Indexes/MapReduce/OutputToCollection/DeleteReduceOutputDocumentsCommand.cs

```diff
@@ -117,7 +122,13 @@ private void CleanupReferenceDocuments(DocumentsOperationContext context, Dictio
 
                     using (var doc = context.ReadObject(referenceDocument.Data, referenceDocument.Id))
                     {
-                        _database.DocumentsStorage.Put(context, referenceDocument.Id, null, doc);
+                        if (doc.TryGet(nameof(OutputReduceToCollectionReference.ReduceOutputs), out BlittableJsonReaderArray updatedIds) == false)
+                            ThrowIdsPropertyNotFound(referenceDocument.Id);
```

Comment:
```suggestion
                            ThrowReduceOutputsPropertyNotFound(referenceDocument.Id);
```

Reply:
Renamed in https://github.com/ravendb/ravendb/pull/10055/commits/5902446451ad63e2740e270632e414237edef4e5